client/lib/plugins/wporg-data/actions.js

```diff
@@ -1,31 +1,34 @@
 /**
  * External dependencies
  */
-var debug = require( 'debug' )( 'calypso:wporg-data:actions' );
+import debugFactory from 'debug';
+import { includes } from 'lodash';
 
 /**
  * Internal dependencies
  */
-var Dispatcher = require( 'dispatcher' ),
-	wporg = require( 'lib/wporg' ),
-	debounce = require( 'lodash/debounce' ),
-	utils = require( 'lib/plugins/utils' );
+import Dispatcher from 'dispatcher';
+import wporg from 'lib/wporg';
+import debounce from 'lodash/debounce';
+import utils from 'lib/plugins/utils';
+import CuratedPlugins from 'lib/plugins/wporg-data/curated.json';
 
 /**
  * Constants
  */
-var _LIST_DEFAULT_SIZE = 24,
-	_DEFAULT_FIRST_PAGE = 0;
+const debug = debugFactory( 'calypso:wporg-data:actions' );
+const _LIST_DEFAULT_SIZE = 24;
+const _DEFAULT_FIRST_PAGE = 0;
 
 /**
  *  Local variables;
  */
-var _fetchingLists = {},
-	_currentSearchTerm = null,
-	_lastFetchedPagePerCategory = {},
-	_totalPagesPerCategory = {};
+let _fetchingLists = {};
+let _currentSearchTerm = null;
+let _lastFetchedPagePerCategory = {};
+let _totalPagesPerCategory = {};
```

Comment:
For folks reviewing, these refs are updated in `reset()`. 

For a future janitorial, I'd prefer if we renamed these without the `_` prefix, and moved this to component or global state. Note that it might just be worth waiting to port this to redux.